src/externalized/ArmourModel.cc

```diff
@@ -31,15 +63,17 @@ ArmourModel::ArmourModel(
 			uint8_t armourClass,
 			uint8_t protection,
 			uint8_t explosivesProtection,
-			uint8_t degradePercentage
+			uint8_t degradePercentage,
+			bool ignoreForMaxProtection
 	) : ItemModel(itemIndex, std::move(internalName), std::move(shortName), std::move(name), std::move(description), IC_ARMOUR, 0, INVALIDCURS, std::move(inventoryGraphics), std::move(tileGraphic), weight, perPocket, price, coolness, reliability, repairEase, flags) {
 	this->armourClass = armourClass;
 	this->protection = protection;
 	this->explosivesProtection = explosivesProtection;
 	this->degradePercentage = degradePercentage;
+	this->ignoreForMaxProtection = ignoreForMaxProtection;
 }
 
-ArmourModel* ArmourModel::deserialize(const JsonValue &json, const BinaryData& vanillaItemStrings) {
+ArmourModel* ArmourModel::deserialize(const JsonValue &json, const BinaryData& vanillaItemStrings, bool extraAttachmentsEnabled) {
```

Comment:
This new parameter is unused.

Reply:
Good catch. I changed the way the policy was evaluated during the dev process. Removed.